clients/src/main/java/org/apache/kafka/common/InvalidRecordException.java

```diff
@@ -14,11 +14,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.kafka.common.record;
+package org.apache.kafka.common;
 
-import org.apache.kafka.common.errors.CorruptRecordException;
+import org.apache.kafka.common.errors.ApiException;
 
-public class InvalidRecordException extends CorruptRecordException {
+public class InvalidRecordException extends ApiException {
```

Comment:
Hmm.. Isn't this an incompatible change? Does this exception not get exposed anywhere currently? Maybe it would be better to deprecate `common/record/InvalidRecordException`.

Reply:
Yeah I think `InvalidRecordException` was not exposed as a public API before, and now we are moving its package as well to make it a public class.